ciao-launcher/instance_test.go

```diff
@@ -209,18 +209,57 @@ func (v *instanceTestState) verifyStatsUpdate(t *testing.T, cmd interface{}) {
 	}
 }
 
-func (v *instanceTestState) expectStatsUpdate(t *testing.T, ovsCh <-chan interface{}) bool {
+func (v *instanceTestState) getStatsUpdate(t *testing.T, ovsCh <-chan interface{}) *ovsStatsUpdateCmd {
 	var cmd interface{}
 	select {
 	case cmd = <-ovsCh:
 	case <-time.After(time.Second):
 		t.Error("Timed out waiting for ovsStatsUpdateCmd")
-		return false
+		return nil
 	}
 	stats, ok := cmd.(*ovsStatsUpdateCmd)
 	if !ok {
 		t.Error("Unexpected Command received on ovsCh")
 	}
+	return stats
+}
+
+func (v *instanceTestState) expectStatsUpdateWithVolumes(t *testing.T,
+	ovsCh <-chan interface{}, volumes []string) bool {
```

Comment:
The bool allows us to detect an error and quit the test case early, shutting down cleanly before we exit.  This is the model used in the similar expectStatsUpdate method and so I duplicated the behaviour when adding the expectStatsUpdateWithVolumes.  However, none of the storage tests do actually quit early if there is an error.  They keep executing and logging any errors they encounter, so as you point out, the return value for expectStatsUpdateWithVolumes isn't currently used.  It may be in the future though, and to be consistent with the expectStatsUpdate method, I'd prefer to keep it.